src/index.js

```diff
@@ -1,3 +1,9 @@
 export { dndzone } from './action.js';
 export { alertToScreenReader } from './helpers/aria';
-export { TRIGGERS, SOURCES, SHADOW_ITEM_MARKER_PROPERTY_NAME, overrideItemIdKeyNameBeforeInitialisingDndZones } from './constants';
+export {
+    TRIGGERS,
+    SOURCES,
+    SHADOW_ITEM_MARKER_PROPERTY_NAME,
+    overrideItemIdKeyNameBeforeInitialisingDndZones,
+    setDebugMode,
```

Comment:
We should probably mention it in the README file and add a declaration to `types/index.d.ts`

Reply:
Done